recipes/libcurl/all/conanfile.py

```diff
@@ -467,151 +503,89 @@ def _arm_version(self, arch):
             version = int(match.group(1))
         return version
 
-    def _build_with_autotools(self):
-        with tools.chdir(self._source_subfolder):
-            # autoreconf
-            self.run("{} -fiv".format(tools.get_env("AUTORECONF") or "autoreconf"), win_bash=tools.os_info.is_windows, run_environment=True)
-
-            # fix generated autotools files to have relocatable binaries
-            if tools.is_apple_os(self.settings.os):
-                tools.replace_in_file("configure", "-install_name \\$rpath/", "-install_name @rpath/")
-
-            self.run("chmod +x configure")
-
-            # run configure with *LD_LIBRARY_PATH env vars it allows to pick up shared openssl
-            with tools.run_environment(self):
-                autotools, autotools_vars = self._configure_autotools()
-                autotools.make(vars=autotools_vars)
-
-    def _configure_autotools_vars(self):
-        autotools_vars = {}
-        # tweaks for mingw
-        if self._is_mingw:
-            autotools_vars["RCFLAGS"] = "-O COFF"
-            if self.settings.arch == "x86":
-                autotools_vars["RCFLAGS"] += " --target=pe-i386"
-            else:
-                autotools_vars["RCFLAGS"] += " --target=pe-x86-64"
-        return autotools_vars
-
-    @functools.lru_cache(1)
-    def _configure_autotools(self):
-        autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
-
-        if self.settings.os != "Windows":
-            autotools.fpic = self.options.get_safe("fPIC", True)
-
-        autotools_vars = self._configure_autotools_vars()
-
-        # tweaks for mingw
-        if self._is_mingw:
-            autotools.defines.append("_AMD64_")
-
-        if cross_building(self) and tools.is_apple_os(self.settings.os):
-            autotools.defines.extend(['HAVE_SOCKET', 'HAVE_FCNTL_O_NONBLOCK'])
-
-        configure_args = self._get_configure_command_args()
-
-        if self.settings.os == "iOS" and self.settings.arch == "x86_64":
-            # please do not autodetect --build for the iOS simulator, thanks!
-            autotools.configure(vars=autotools_vars, args=configure_args, build=False)
```

Comment:
Not migrated. Needed ?

Reply:
The new [Autotools build helper](https://docs.conan.io/en/latest/reference/conanfile/tools/gnu/autotools.html) doesn't have a build argument.